### DISCUSSION TITLE
A Discord Server can be quite useful inorder to discuss something.

### DISCUSSION THREAD

**Discussion by ashikpy** [LazyVim/LazyVim#4417] [Recipes]
A Discord server might be useful

**Comment by abe-101** [comment]
Does LazyVim have any social channels besides GitHub discussions?

**Reply by folke** [reply]
Nope, I have too many channels to monitor already.
There's a matrix room though, but I'm not really active there myself. See the website for details.

**Reply by dpo** [reply]
On mobile, I simply cannot find it.

**Reply by dpo** [reply]
After checking the desktop site, it turns out the Matrix link is in the top bar, which is not visible on the mobile site :-(.

**Comment by DanRioDev** [comment]
Discord is not indexable, solutions get lost

**Comment by ThallesP** [comment]
@folke can you re-consider? discord would be so good, lunarvim has one and I got a faster response there, gh discussions sucks to write and get a response.
for indexing solutions, use something like https://www.answeroverflow.com/ where it does the job.
